### DISCUSSION TITLE
Jandy - Direct Pump Control

### DISCUSSION THREAD

**Discussion by nickhudkins** [aqualinkd/AqualinkD#188] [Q&A]
Howdy folks!

I have recently gotten a Century V-Green 165 to replace a Jandy ePump. (JEP-1.5)

I had a JEP-R controller: https://www.jandy.com/en/products/pool-pumps/jep-r which is connected and working thanks to some help from this thread: https://www.troublefreepool.com/threads/century-regal-vgreen-motor-automation.238733/

I also have an AquaPure (APUREM) SWG, but precisely ZERO pool automation beyond the JEP-R and the APUREM front panel (Chlorine Production + and -, pool temp, and salinity reading)

It appears that much of the AqualinkD project is intended to automate... the Aqualink (automation system) and was curious if anyone has used this in a more direct manner.

I've done some logging of serial commands with the JEP-R connected, but have yet to parse the HEX commands to understand their makeup.

It seems Jandy is a bit.... customer-hostile as far as sharing automation support, so looking for any head start I can get so I'm not re-inventing the wheel here.

**Comment by mguinness** [comment]
If you haven't already seen it take a look at [this thread](https://www.troublefreepool.com/threads/jandy-pump-protocol.265447/) on TFP.
